libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
one solution is to provide a test support package. e.g [net/http/httptest](https://golang.org/pkg/net/http/httptest/).